lib/dl_api_lib/dl_api_lib/dataset/validator.py

```diff
@@ -1147,7 +1147,6 @@ def apply_source_action(
     ) -> None:
         """Apply update to the data source configuration"""
 
-        self._sync_us_manager.load_dependencies(self._ds)
```

Comment:
This raises an exception from USEntryBuffer in case the connection could not be loaded previously, so removing it – the connection will be loaded anyway when we first try to access it and the exception is handled there